react/features/toolbox/components/web/Toolbox.js

```diff
@@ -976,29 +975,33 @@ class Toolbox extends Component<Props, State> {
                 visible = { this._shouldShowButton('settings') } />,
             this._shouldShowButton('stats')
                 && <OverflowMenuItem
-                    accessibilityLabel =
-                        { t('toolbar.accessibilityLabel.speakerStats') }
+                    accessibilityLabel = { t('toolbar.accessibilityLabel.speakerStats') }
                     icon = { IconPresentation }
                     key = 'stats'
                     onClick = { this._onToolbarOpenSpeakerStats }
                     text = { t('toolbar.speakerStats') } />,
             this._shouldShowButton('feedback')
                 && _feedbackConfigured
                 && <OverflowMenuItem
-                    accessibilityLabel =
-                        { t('toolbar.accessibilityLabel.feedback') }
+                    accessibilityLabel = { t('toolbar.accessibilityLabel.feedback') }
                     icon = { IconFeedback }
                     key = 'feedback'
                     onClick = { this._onToolbarOpenFeedback }
                     text = { t('toolbar.feedback') } />,
             this._shouldShowButton('shortcuts')
                 && <OverflowMenuItem
-                    accessibilityLabel =
-                        { t('toolbar.accessibilityLabel.shortcuts') }
+                    accessibilityLabel = { t('toolbar.accessibilityLabel.shortcuts') }
                     icon = { IconOpenInNew }
                     key = 'shortcuts'
                     onClick = { this._onToolbarOpenKeyboardShortcuts }
-                    text = { t('toolbar.shortcuts') } />
+                    text = { t('toolbar.shortcuts') } />,
+            this._shouldShowButton('help') && typeof _helpLink === 'string'
+                && <OverflowMenuItem
```

Comment:
Please wrap this in a button. See: https://github.com/jitsi/jitsi-meet/blob/master/react/features/etherpad/components/SharedDocumentButton.js (check git history on that file to see how the Toolbox got simplified)

Reply:
ok. although i'm not a big fan of introducing new component for something so simple and not reusable...

Reply:
fixed!

Reply:
FTR: it is **very** reusable. We get to share 100% of the code across web and mobile. This is not my own preference but an overall goal we have as a team.